tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+# Evaluation of Microsoft Playwright
+
+## Introduction
+**Automation testing**
+Automation testing is a way to automate one's browser by simulating user actions like clicks. As Software or applications become more complex as they are built on layers and entire networks of sub-systems, including UI and API layers, external databases, networks, and even third-party integrations, there is always a need for thorough testing to be done, this ranges from unit testing to end to end testing of the application. Browser automation frameworks to perform end-to-end testing helps to ascertain the stability and the health of the application which in turn leads to the success of the application when on production. The general stability of the application doesn't entirely depend on end-to-end testing but it surely helps detects bugs, among other benefits, thus increasing application productivity. Below are some of the popular test automation frameworks;
+- Selenium
+- Puppeteer
+- Cypress
+- Playwright(maybe the new kid on the block but it's gaining quite a trajectory among Software developers and testers)
+
+### Project Context
+The Playwright evaluation is a result of a need for potential replacement of [WebdriverIO](https://webdriver.io/) with one of the best non-selenium modern automation testing framwork. Wikimedia supports a large number of repositories as well as extensions. In order to ensure good code practices across all these repositories, an extensive amount of testing is performed and one of the tests performed is an end to end test. WebdriverIO is the current browser automation framework being used for implementing the end-to-end tests. However, with the recent increase in awareness about end-to-end testing, a number of equally competitive non-selenium solutions have been introduced and one of them is [Microsoft Playwright](https://github.com/microsoft/playwright).
+
+At the time of choosing which automation framework to use in the implementation of the automation testing, extensive research was done and an [evaluation of WebdriverIO](https://filipin.eu/selenium-javascript) was carried out by Željko Filipin, who is a Senior Engineer in Test at Wikimedia Foundations and also one of my mentors. In search for a potential replacement of WebdriverIO, a number of modern and non-selenium automation frameworks have been evaluated against WebdriverIO. These evaluations were done by Soham who was Google Summer of Code 2020 intern and one of my mentors, they can be found using the links below;
```

Comment:
I would mention Soham's last name too. 😄

Reply:
Just leaving a note here, we are not in a need for a potential replacement. WebdriverIO is really powerful but we just want to ensure we are using the right tools that make testing easy for us to maintain and our developers to write. Might want to make that bit sound appropriately 😁